Resources/Prototypes/_EinsteinEngines/Species/ipc.yml

```
@@ -174,7 +201,9 @@
 - type: marking
   id: MobIPCLLegDefault
   bodyPart: LLeg
-  markingCategory: Legs
+  markingCategory: LeftLeg
+  speciesRestriction: [ IPC ]
+  markingCategory: RightLeg
```

YAMLlint (1.37.1)

[error] duplication of key "markingCategory" in mapping

(key-duplicates)